doc/rados/operations/placement-groups.rst

```diff
@@ -664,31 +667,33 @@ Valid formats are ``plain`` (default) and ``json``.
 Get Statistics for Stuck PGs
 ============================
 
-To get the statistics for all placement groups stuck in a specified state,
-execute the following:
+To see the statistics for all PGs that are stuck in a specified state, run a
+command of the following form:
 
 .. prompt:: bash #
 
    ceph pg dump_stuck inactive|unclean|stale|undersized|degraded [--format <format>] [-t|--threshold <seconds>]
 
-**Inactive** Placement groups cannot process reads or writes because they are waiting for an OSD
-with the most up-to-date data to come up and in.
+- **Inactive** PGs cannot process reads or writes because they are waiting for
+  an OSD with the most up-to-date data to come ``up`` and ``in``.
 
-**Unclean** Placement groups contain objects that are not replicated the desired number
-of times. They should be recovering.
+- **Unclean** PGs contain objects that have not been replicated the desired
+  number of times. Under normal conditions, it can be assumed that these PGs
+  are recovering.
```

Comment:
Actually I think this describes **Undersized**.  `Unclean` is more nuanced.  If we aren't going to explain all the states here, I suggest we leave out `unclean`.

Reply:
I will cut it out.

Accepted.

Reply:
To be clear I was suggesting leaving the description in and changing s/**Unclean**/**Undersized**/